fonts.css

```diff
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: "Roboto";
+  src: url("fonts/Roboto-Black.woff") format("woff");
+  src: url("fonts/Roboto-Black.woff2") format("woff2");
+  font-weight: 900;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("fonts/Roboto-Medium.woff") format("woff");
+  src: url("fonts/Roboto-Medium.woff2") format("woff2");
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6245874)